rust/parallel_calls/src/callee/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "callee"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+candid = "0.10"
+ic-cdk = "0.13"
+ic-cdk-timers = "0.7" # Feel free to remove this dependency if you don't need timers
```

Comment:
Do you need timers? I didn't see anything in the source files.

Reply:
Nope, thanks!